spot_driver/launch/spot_driver.launch.py

```diff
@@ -106,6 +109,14 @@ def launch_setup(context: LaunchContext, ld: LaunchDescription) -> None:
 
     ld.add_action(rviz)
 
+    depth_images = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [PathJoinSubstitution([FindPackageShare(THIS_PACKAGE), "launch", "spot_depth_publishers.launch.py"])]
+        ),
+        launch_arguments={"spot_name": spot_name, "camera_sources_yaml": camera_sources_yaml}.items(),
+    )
+    ld.add_action(depth_images)
```

Comment:
Could you add a launch argument to toggle between using the new separate registered depth nodelets and using the previous approach of requesting registered depth images through the Spot SDK? It should default to `true`, and if it's `true` it should disable the parameter in the spot_ros2 driver node to request registered depth images. If it's `false` the opposite should happen -- the new nodelets should not be brought up, and if the `publish_depth_registered` parameter is `true` it should be passed in to the spot_ros2 driver node as usual.